docs/program/sdks/_index.md

```diff
@@ -204,6 +205,43 @@ async function main() {
 main();
 ```
 
+{{% /tab %}}
+{{% tab name="C++" %}}
```

Comment:
Noticed: selecting one tab on page does not auto-switch to same-name tabs everywhere on page. I can look into this!